sql/migrations/20210727201033_world.sql

```diff
@@ -75,7 +75,7 @@ INSERT INTO `spell_target_position` (`id`, `target_map`, `target_position_x`, `t
 (26456, 0, 1642.41, 239.9, 62.59, 3.01, 5086, 5875);
 
 -- Lunar Festival Sentinel texts are chat_type 12.
-UPDATE `broadcast_text` SET `chat_type`='12' WHERE `entry` IN (10948,10949,10950,10951,10952,10953,10954,10955);
+UPDATE `broadcast_text` SET `chat_type`=12 WHERE `entry` IN (10948,10949,10950,10951,10952,10953,10954,10955);
```

Comment:
Chat type 12??? The maximum is 6.
https://github.com/vmangos/core/blob/808ec9801870ae926fa037f100bca4d2c3efa2d8/src/game/Objects/CreatureDefines.h#L383